contactsummary/tests/phpunit/CRM/Contactsummary/Utils/ContractSummaryTestTrait.php

```diff
@@ -94,6 +94,12 @@ protected function requestLeave($absenceTypeID, $contactID, $from, $to, $absence
     $mainActivity = civicrm_api3('Activity', 'create', $activityParam);
     $sourceActivityID = $mainActivity['id'];
 
+    civicrm_api3('ActivityContact', 'get', array(
+      'activity_id' => $sourceActivityID,
+      'contact_id' => $contactID,
+      'record_type_id' => 3,
+    ));
```

Comment:
Oh it should be create not get ... ( I am still running the tests (they take so much :( ) by the way and I was not planning to merge before they pass  )